modules/fakturabank/model/fakturabank.class.php

```diff
@@ -1605,6 +1606,10 @@ public function registeroutgoing() {
                     $fbvoting = new lodo_fakturabank_fakturabankvoting();
                     $fbvoting->update_fakturabank_outgoing_invoice($InvoiceO->FakturabankID, $ID, $InvoiceO->AccountPlanID);
 
+                    # blank update to set invoice sender data
+                    $tmp_invoice = new invoice(array('CustomerAccountPlanID' => $InvoiceO->AccountPlanID, 'VoucherType' => 'S', 'InvoiceID' => $ID));
+                    $tmp_invoice->update(array('InvoiceID' => $ID, 'invoiceout_DueDate_'.$ID => $InvoiceO->PaymentMeans->PaymentDueDate, 'invoiceout_CustomerAccountPlanID_'.$ID => $InvoiceO->AccountPlanID));
+
```

Comment:
Check if this should update all data for supplier (company that we are downloading for).
Example: We use 2 bank account, if we use them for different invoices, all of them will be overwritten to default one when we download. We will not use XML data (that is wrong I think).